src/Miningcore/Payments/PaymentSchemes/PPSPaymentScheme.cs

```diff
@@ -291,6 +299,9 @@ private async Task<decimal> CalculateBlockData(decimal blockRewardInEth, PoolCon
                 valPerHash = blockData / (Decimal) sumDifficulty;
             }
             poolConfig.PaymentProcessing.HashValue = valPerHash;
+
+            // Update the hashvalue in the database
+            Task.WaitAll(cf.Run(con => paymentRepo.SetPoolStateHashValue(con, poolConfig.Id, (double) valPerHash)));
```

Comment:
Double precision will cut off some of our decimal places.   Is it possible to keep this as a Decimal?  Or use just a string?